### ISSUE TITLE
Trigger server restart create/delete language settings manager plugin

### ISSUE THREAD

**Issue by forsetius** [strapi/strapi#838]
**Node.js version**:
v9.9.0
**npm version**:
5.6.0
**Strapi version**:
3.0.0-alpha11.1
**Operating system**:
Linux Mint 18.3 Cinnamon 64-bit

**Do you want to request a *feature* or report a *bug*?**
Bug
**What is the current behavior?**
Deleting the language in Settings Manager has no long-term effect. It seems to be deleted but after refresh they are there again. Trying to add a language redirects to empty page.

**If the current behavior is a bug, please provide the steps to reproduce the problem**
In Admin > Settings Manager > Languages:
One by one, I delete all the languages except default (English). After each deletion given language disappears and the console produces messages like:
```
debug DELETE /settings-manager/configurations/languages/it_it (10 ms)
info File changed: /home/username/node/site/config/locales/it_it.json
info The server is restarting
```

Wait until server reloads, then refresh the page. The deleted languages are listed again. However, now their locales are `it_it` instead of `it_IT`. Looking into `./config/locales` you'll find two sets of files, for example `it_IT.json` with `{"welcome": "Benvenuto"}` inside and `it_it.json` with `{}` inside.

**What is the expected behavior?**
When server is restarting there should be modal with rotating arrows. In other cases when server reloads it is there but not after deleting a language. Then, if I delete a language it should be deleted and not visible on the list anymore. Also, UI for adding a language should work.

**Comment by lauriejim** [comment]
Yes you are right.

It's not an hard issue to fix. I think you can fix it and submit a PR.
If you have any question on how to did it, maybe @soupette can help you.

**Comment by forsetius** [comment]
Well, I tried it but it's not enough.
In `./plugins/settings-manager/controllers/SettingsManager.js` at line 149 (`deleteLanguage` function just before closing `}`) I've added:
```js
strapi.reload();
```
and it triggers server restart but after refreshing the page, deleted languages are back there. It seems that somehow `createLanguage` function is called with its
```js
fs.writeFileSync(filePath, '{}');
```

BTW: I've found another bug while trying to fix this one. It's about translation labels - I'll issue PR shortly.
